src/server/src/modules/message.ts

```diff
@@ -1,7 +1,8 @@
 import { Message } from '@entities/message';
 import { User } from '@entities/user';
 import { getRepository } from 'typeorm';
-import { userIdExists } from '@modules/user';
+import { userIdExists, getUserById } from '@modules/user';
+import { sendEmail } from '@modules/mail';
```

Comment:
import from lib after mail file is moved

Reply:
Done.